erica/infrastructure/sqlalchemy/alembic/versions/5313b3c6d8ec_added_erica_request_table.py

```diff
@@ -16,7 +16,9 @@
 depends_on = None
 
 
-def upgrade():
+def upgrade():    
+    op.execute('DROP TYPE IF EXISTS requesttype')
```

Comment:
Why are we dropping here the enum types? In case they were already there? Shouldn't we then also drop the table in case it is there?